src/elements/Segment/Segments.js

```diff
@@ -0,0 +1,46 @@
+import React, {Children, Component, PropTypes} from 'react';
+import _ from 'lodash';
+import invariant from 'fbjs/lib/invariant';
+import classNames from 'classnames';
+import META from 'src/utils/Meta';
+import Segment from './Segment';
+
+export default class Segments extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+  };
+
+  componentDidMount() {
+    const {children} = this.props;
+
+    invariant(
+      !_.any( Children.map(children, child => child.type !== Segment) ),
+      'May only contain children of type Segment.',
+    );
```

Comment:
In attempting to allow `Segments` to support nested `Segments` I tried the propTypes approach, as well as the following without luck:

```
invariant(
  !_.any(Children.map(children, child => {
    return _.includes( ['Segment', 'Segments'], _.get(child, 'type._meta.name') );
  })),
  'May only contain children of type Segment or Segments.',
);
```
As a result, I'm going to backpedal to `children: PropTypes.node` for now, and remove the `fbjs` and `invariant` calls. The net result is the component will accept any children until we're able to figure out a method which works.